config/main.py

```diff
@@ -3534,6 +3534,34 @@ def delete(ctx):
 
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
+@config.group(cls=AbbreviationGroup)
+@click.pass_context
+def switch(ctx):
+    """switch related config"""
+    pass
+
+@switch.command('shell')
+@click.pass_context
+@click.argument('en', metavar='<en>', required=True, type=int)
```

Comment:
How the CLI will look? will it be en=0 or en=1?

Reply:
will change it to click.choice